WORKSPACE

```diff
@@ -375,6 +375,13 @@ http_file(
 )
 
 # some repos which are not part of go_rules anymore
+go_repository(
+    name = "org_golang_x_sys",
+    importpath = "golang.org/x/sys",
```

Comment:
Shouldn't this belong into `go.mod`?

Reply:
under normal circumstances (no bazel) you're totally right but there's a rabbit hole since gazelle and other things in WORKSPACE depend on their own versions of golang/x/y

Reply:
We did it for main but it continues to be a problem https://github.com/kubevirt/kubevirt/pull/12617